requirements/run.in

```diff
@@ -0,0 +1 @@
+requests==2.27.0
```

Comment:
`httpx` being a core `kytos` dependency is encouraged to be used instead. `requests`  current usage should only be for old related code, otherwise you'd be introducing a new dependency that's not aligned with `asyncio` usage and how the platform and NApps are evolving.

Reply:
Deleted `requests`, now using `httpx`